docs/commonly_used_data_types/asset_selection.md

```diff
@@ -16,4 +16,14 @@ asset_selection: {
 }
 ```
 
-NOTE: The machine_type strings must match source_type, NOT source_type_clean.
\ No newline at end of file
+~~NOTE: The machine_type strings must match source_type, NOT source_type_clean.~~
+
+NOTE: Now we can use source_type_clean as well in asset_selection like below for KPIs related SDK functions-
```

Comment:
```suggestion
NOTE: We can use source_type_clean as well in asset_selection like below for KPIs related SDK functions-
```
IMO these docs should alway be written with present tense and the idea that it's being read fresh and only applies to the current code.  We can add a change log if wanted for noting changes specifically.

Reply:
Commit your changes